client/my-sites/activity/filterbar/text-selector.tsx

```diff
@@ -20,6 +20,14 @@ const TextSelector: FunctionComponent< Props > = ( { siteId, filter } ) => {
 	const isMobile = useMobileBreakpoint();
 
 	const [ searchQuery, setSearchQuery ] = useState( filter.textSearch || '' );
+
+	useEffect( () => {
+		// If the filter is cleared, clear the search query
```

Comment:
This is to force clearing the search query text when clicking on `Remove all filters`.